src/mapstorage/XmlMapStorage.cpp

```diff
@@ -155,6 +155,10 @@ XmlMapStorage::Converter::Converter()
             if (str == "UNDEFINED") {
                 str.clear(); // we never save or load the string "UNDEFINED"
             } else {
+                if (str == "EXIT") {
+                    // we save the EXIT flag inverted => invert the name too
+                    str = "NO_EXIT";
```

Comment:
Is NO_EXIT even needed? What's the difference between "north" and "south" in the below example:

```
        <exit dir="north">
            <to>1918</to>
            <exitflag>NO_EXIT</exitflag>
        </exit>
        <exit dir="south"/>
```

Reply:
The `NO_EXIT` flag - which corresponds to the lack of `EXIT` flag in C++ - is used to mark exceptional exits,
which appear in red in the map.
It indicates places where you can exit in such direction due to external events, but is not an exit you can **decide** to take.
For example, the river rapids sending you to the shore just outside Lorien forest south border.